Traceback (most recent call last):
  File "/home/vasu/Work/pokemon_ai/showdown_ai/showdown.py", line 231, in run
    self.play_game(challenge=challenge)
  File "/home/vasu/Work/pokemon_ai/showdown_ai/showdown.py", line 210, in play_game
    self.selenium.switch(move.switch_index, move.backup_switch)
  File "/home/vasu/Work/pokemon_ai/showdown_ai/browser.py", line 188, in switch
    buttons[i].click()
IndexError: list index out of range
